lib/solid/parser.ex

```diff
@@ -399,6 +399,20 @@ defmodule Solid.Parser do
     |> ignore(closing_tag)
     |> tag(:continue_exp)
 
+  raw_tag =
+    ignore(opening_tag)
+    |> ignore(space)
+    |> ignore(string("raw"))
+    |> ignore(space)
+    |> ignore(closing_tag)
+    |> repeat(lookahead_not(parsec(:end_raw_tag)) |> utf8_char([]))
+    |> ignore(opening_tag)
```

Comment:
Maybe you can replace the following `ignore`s by a single `ignore(parsec(:end_raw_tag))`. What do you think?

Reply:
BTW I think we don't need a `defcombinatorp` here as there's no recursion. Maybe a simple variable will do the job?

```
  end_raw_tag =
    opening_tag
    |> ignore(space)
    |> ignore(string("endraw"))
    |> ignore(space)
    |> ignore(closing_tag)

  raw_tag =
    ignore(opening_tag)
    |> ignore(space)
    |> ignore(string("raw"))
    |> ignore(space)
    |> ignore(closing_tag)
    |> repeat(lookahead_not(ignore(end_raw_tag)) |> utf8_char([]))
    |> ignore(end_raw_tag)
    |> tag(:raw_exp)
```

Reply:
I'm sure I tried this and it didn't work, but I must have been doing something wrong. Your example worked fine. Thanks

Reply:
TBH I don't understand fully how `lookahead_not` works. It's a bit of trial and error 🙊